src/edu/csus/ecs/pc2/core/imports/clics/CLICSJsonUtilities.java

```diff
@@ -0,0 +1,506 @@
+import java.util.logging.Level;
+
+import javax.xml.bind.JAXBException;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.csus.ecs.pc2.core.Utilities;
+import edu.csus.ecs.pc2.core.exception.IllegalContestState;
+import edu.csus.ecs.pc2.core.list.RunComparatorByElapsedRunIdSite;
+import edu.csus.ecs.pc2.core.log.StaticLog;
+import edu.csus.ecs.pc2.core.model.Account;
+import edu.csus.ecs.pc2.core.model.ClientId;
+import edu.csus.ecs.pc2.core.model.ClientType;
+import edu.csus.ecs.pc2.core.model.ElementId;
+import edu.csus.ecs.pc2.core.model.FinalizeData;
+import edu.csus.ecs.pc2.core.model.Group;
+import edu.csus.ecs.pc2.core.model.IInternalContest;
+import edu.csus.ecs.pc2.core.model.Problem;
+import edu.csus.ecs.pc2.core.model.Run;
+import edu.csus.ecs.pc2.core.security.Permission.Type;
+import edu.csus.ecs.pc2.core.standings.ContestStandings;
+import edu.csus.ecs.pc2.core.standings.ScoreboardUtilites;
+import edu.csus.ecs.pc2.core.standings.TeamStanding;
+import edu.csus.ecs.pc2.core.standings.json.ScoreboardJsonModel;
+import edu.csus.ecs.pc2.core.standings.json.StandingScore;
+import edu.csus.ecs.pc2.core.standings.json.TeamScoreRow;
+
+/**
+ * A set of methods for CLICS API JSON
+ * 
+ * @author Douglas A. Lane <pc2@ecs.csus.edu>
+ *
+ */
+public class CLICSJsonUtilities {
+
+    public static final String GROUP_WINNER_TITLE = "group-winner-";
+
+    public static final String WINNER_S_OF_GROUP_TITLE = "Winner(s) of group ";
+
+    public static final String ID_WINNER = "winner";
+
+    public static final String ID_BRONZE_MEDAL = "bronze-medal";
+
+    public static final String ID_SILVER_MEDAL = "silver-medal";
+
+    public static final String ID_GOLD_MEDAL = "gold-medal";
+
+    private static ObjectMapper mapperField;
+
+    // {"id":"group-winner-4","citation":"Winner(s) of group Eindhoven University of Technology","team_ids":["28"]},
+
+//  "citation": "Winner(s) of group Delft University of Technology", 
+// "citation": "Winner(s) of group Eindhoven University of Technology", 
+//
+//
+// "citation": "First to solve problem crashingcompetitioncomputer", 
+// "citation": "First to solve problem grindinggravel", 
+// "citation": "First to solve problem housenumbering", 
+//
+// "citation": "Contest winner",     
+
+    /**
+     * Return a list of CLICS Awards for a contest.
+     * 
+     * @param contest
+     * @return
+     * @throws IOException
+     * @throws IllegalContestState
+     * @throws JAXBException
+     * @throws JsonMappingException
+     * @throws JsonParseException
+     */
+    public static List<CLICSAward> createAwardsList(IInternalContest contest) throws JsonParseException, JsonMappingException, JAXBException, IllegalContestState, IOException {
```

Comment:
Why does this method declare that it `throws JsonParseException` and `JsonMappingException`?   There's nothing in the method (or things it indirectly calls) that can throw those exceptions.  Methods shouldn't declare that they `throw` something that the don't/can't.  Recommend deleting these two exceptions from the `throws` clause.